plugins/org.eclipse.n4js.model/emf-gen/org/eclipse/n4js/n4JS/MemberAccess.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright (c) 2016 NumberFour AG.
```

Comment:
this is EMF-generated code; we can't bump the copyright year for individual EMF entities